content/tutorials/wallets.md

```diff
@@ -3,81 +3,106 @@ title: Wallets for Ocean Tokens
+As the name suggests, a crypto wallet is a type of digital wallet that stores cryptocurrencies. It uses private and public key based encryption to allow users to send and receive cryptocurrencies.
 
-An account is identified by its address, so if you want someone to send some Ocean Tokens to a specific account, you give them the account's address.
+### Why do we need a wallet?
 
-The private key is used to spend the Ocean Tokens in the account. You must keep it secret, because anyone with that private key can spend those tokens. If you lose the secret key, and nobody else has it, then _nobody_ can spend those tokens, so don't lose it!
+Unlike traditional pocket wallets, digital wallets do not actually store the currency. For example, your bank app or website does not store the money. It just shows what is recorded on the bank’s servers. The difference for a cryptocurrency is that the record of your transactions are written onto the blockchain, so everyone who is a part of the blockchain has access to these transaction records. A crypto wallet is a software program that interacts with these various blockchains to show the final recorded amount so the wallet owners can monitor their balance, and send/receive cryptos.
 
-> Note: The same account might have an Ocean Token balance, an [Ether](https://www.ethereum.org/ether) balance, and other balances.
+### What are the different types of wallets?
+
+Just as wallets come in various shapes and sizes in the real world, several types of crypto-wallets are available, offering different ways of storing your cryptocurrencies. Each type has its own strengths and weaknesses.
+
+<!-- prettier-ignore-start -->
+| Type of Wallet | Strengths | Weaknesses
+| --- | --- | --- |
+| Desktop/mobile | These are downloaded and stored on the desktop or mobile device and can only be accessed through this one device so it&#39;s highly secure. Mobile wallets have the additional benefit of being portable | If the computer/phone is hacked or gets a virus, there's a possibility that your funds could be lost forever. Mobile wallets are also restricted by limited space so tend to be smaller.
+| [Hardware](#hardware-wallets) | The private keys are stored on a physical device such as a USB stick. Hardware wallets have the ability to make transactions online, but are very secure because they are stored offline. | If you lose your hardware wallet and its associated seed phrase, all your funds are lost. So this may not be the best option for careless folks!
+| [Software](#software-wallets) | These are cloud-based wallets that are accessible over the internet. They are the most convenient to access, and are currently the most popular way to store cryptos. | Since online wallets are controlled by third parties, this centralization leads to a greater risk of your private keys being stolen through hacking. |   |
```

Comment:
I think this needs some more differentiation. Current copy makes it sounds like all software wallets are "cloud-based", which to me implies a custodian wallet like Coinbase, blockchain.com, etc. But software wallets like exodus.io, MetaMask are non-custodian (user has private key, and needs to take care of it) so nothing here is cloud-based.

Maybe we split this up into 2 rows, `Custodian Software` & `Non-Custodian Software`, where the latter is then linked to our `#software-wallets` section? Other ideas?

Reply:
ah wait, in the context of the full table: probably is enough to simply rename `Software` to `Online` and not link this one to our section, but the `Desktop/mobile`